test/Spec.hs

```diff
@@ -141,6 +146,80 @@ jsonRoundTrip = describe "JSON Roundtripping" $ do
             Left err -> expectationFailure $ T.unpack err
             Right _ -> pure ()
 
+--------------------------------------------------------------------------------
+-- Evaluation Semantics
+
+instance Arbitrary ValueExt where
+  arbitrary =
+    oneof
+      [ genString,
+        genNumber,
+        genBoolean,
+        genNull,
+        genArray,
+        genObject
+      ]
+    where
+      genString = fmap (String P.emptySpan) arbitrary
+      genNumber = fmap (Number P.emptySpan) arbitrary
+      genBoolean = fmap (Boolean P.emptySpan) arbitrary
+      genArray = resize 6 $ fmap (Array P.emptySpan . V.fromList) $ listOf arbitrary
+      genObject = resize 5 $ fmap (Object P.emptySpan . Compat.fromList) $ listOf arbitrary
+      genNull = pure $ Null P.emptySpan
```

Comment:
💯

Reply:
The Arbitraries are getting so cool Need a twitter bot to tweet a new kriti expression every day 👍